packages/ui-toolkit/src/components/atoms/Button/button.css

```diff
@@ -1,116 +1,164 @@
+.btn96XLargeButton {
+ width: fit-content;
+ height: fit-content;
+ padding: 16px 24px;
+ height: 56px;
+ gap: 8px;
 }
 
-.btn51RippleSecondary:active {
-  background-color: var(--gray700);
-  background-size: 100%;
-  transition: background 0s;
+.btn86FullWidth{
+  width: 100%;
 }
 
-.btn51RippleTertiary {
-  background-position: center;
-  transition: background 0.8s;
+.btnIconLeading {
+  width: 100%;
+  height: 100%;
 }
 
-.btn51RippleTertiary:hover {
-  background: rgba(0, 208, 156, 0.04);
+
+/* -- Seconday Button -- */
+.btn96SecondaryButtonWithAccent {
+  background-color: var(--green100);
   color: var(--green500);
 }
 
-.btn51RippleTertiary:active {
-  background-color: var(--gray700);
-  background-size: 100%;
-  transition: background 0s;
+.btn96SecondaryButtonWithoutAccent {
+  background-color: var(--dangerouslySetPrimaryBg);
+  color: var(--gray900);
+  border: 1px solid var(--gray150);
 }
 
-.btn51Primary {
-  background: var(--green500);
-  color: var(--white);
+.btn96SecondaryButtonWithoutAccent:hover {
+  background-color: var(--gray50);
 }
 
-.btn51Secondary {
-  border: 1px solid var(--green500);
-  color: var(--green500);
+.btn96SecondaryButtonWithoutAccentDisabled {
+  background-color: var(--gray150);
 }
 
-.btn51Tertiary {
-  background: var(--green100);
-  color: var(--green500);
+/* -- Tertairy Button -- */
+
+
+.btn96TertiaryButtonWithoutAccent .btn96ParentDimension {
+  border-bottom: 1px dashed var(--gray700);
 }
 
-.btn51Loading {
-  opacity: 0.5;
-  pointer-events: none;
+.btn96TertiaryButtonWithoutAccent:hover {
+  background-color: var(--gray50);
 }
 
-.btn51DisabledPrimary {
-  background-color: var(--gray500);
-  pointer-events: none;
-  color: var(--white);
+/* -- Compact classes -- */
+
+.btn96CompactButton {
+  width: fit-content;
+  min-width: fit-content;
+  height: fit-content;
+  border-radius: 0;
+  padding: 0 !important;
 }
 
-.btn51DisabledSecondary {
-  border-color: var(--gray500);
-  color: var(--gray500);
-  pointer-events: none;
+
+/* -- Disabled State -- */
+
+.btn96ButtonDisabled {
+  background-color: var(--gray50);
+  border: 1px solid var(--gray50);
 }
 
-.btn51DisabledTertiary {
-  color: var(--gray500);
-  background-color: var(--gray150);
-  pointer-events: none;
+
+.btn96TertiaryButtonDisabled .btn96ParentDimension {
+  border-bottom: 1px dashed var(--gray400);
 }
 
-.btn51BottomFixed {
+
+/* Loader Classes */
+
+.btn96LoadingButton {
+  color: transparent;
+}
+
+.btn96LoadingButton .btn96ParentDimension {
+  border-bottom: none;
+}
+
+.btn96LoaderContainer {
+  position: absolute;
+}
+
+.btn96PrimaryButtonLoader {
+  border: 2px solid var(--dangerouslySetPrimaryBg);
+  border-top-color: transparent;
+}
+
+.btn96LoaderWithoutAccent {
+  border: 2px solid var(--gray900) !important; 
+  border-right-color: transparent !important; 
+}
+
+.btn96MediumButton .btn96LoaderSize {
+  margin-top: 4px;
+  width: 20px; 
+  height: 20px;
+}
+
+.btn96LargeButton .btn96LoaderSize {
```

Comment:
```
.btn96LoaderSize {
     margin-top: 4px;
}
```
This is repeated. Can take this out and write it once

Reply:
fixed